.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: julia
 os:
     - linux
-    - osx
```

Comment:
What I would do is just comment out this line and add a comment with a reference to this PR. That way it's clearer why it was removed.